src/resources/Catalogs/CatalogConfiguration.ts

```diff
@@ -0,0 +1,33 @@
+import API from '../../APICore';
+import {New, PageModel} from '../BaseInterfaces';
+import Resource from '../Resource';
+import {CatalogConfigurationModel, CatalogsListOptions, CreateCatalogConfigurationModel} from './CatalogInterfaces';
+
+export default class CatalogConfiguration extends Resource {
```

Comment:
It doesn't seem to be instantiated anywhere 🤔

Reply:
yup, just pushed it